snmp/datadog_checks/snmp/data/default_profiles/hp-h3c-switch.yaml

```diff
@@ -0,0 +1,45 @@
+extends:
+  - _generic-if.yaml
+  - _hp.yaml
+
+sysobjectid:
+  - 1.3.6.1.4.1.25506.11.1.*
+
+metrics:
+  - MIB: HH3C-ENTITY-EXT-MIB
+    table:
+      OID: 1.3.6.1.4.1.25506.2.6.1.4.1
+      name: hh3cProcessTable
+    symbols:
+      - OID: 1.3.6.1.4.1.25506.2.6.1.4.1.1.3
+        name: hh3cProcessUtil5Min
+    metric_tags:
+      - column:
+          OID: 1.3.6.1.4.1.25506.2.6.1.4.1.1.1
+          name: hh3cProcessID
+        tag: hh3c_process_id
+      - column:
+          OID: 1.3.6.1.4.1.25506.2.6.1.4.1.1.2
+          name: hh3cProcessName
+        tag: hh3c_process_name
+  - MIB: HH3C-ENTITY-EXT-MIB
+    table:
+      OID: 1.3.6.1.4.1.25506.2.6.1.1.1
+      name: hh3cEntityExtStateTable
+    symbols:
+      - OID: 1.3.6.1.4.1.25506.2.6.1.1.1.1.6
+        name: cpu.usage
+      - OID: 1.3.6.1.4.1.25506.2.6.1.1.1.1.8
+        name: memory.usage
```

Comment:
We need to tag by `cpu` and `mem`.  I think you need to split the memory and cpu metrics into 2 tables, then tag the cpu.usage metric with `cpu` and `memory.usage` with `mem`.